### PR TITLE
Added tests for AlertView and added a few more tests for SpatialView.

### PR DESCRIPTION
**Pull request by mbucknell**
I'm going to declare this ticket done. To do's would be refactoring wfs.js and improving the test coverage.

### CODE REVIEW

File: src/test/js/specs/AlertView_spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+	beforeEach(function() {
+		$('body').append('<div id="test-div"></div>');
+	});
+
+	afterEach(function() {
+		$('#test-div').remove();
+	});
+
+	it('Expects that initialize creates the view but does not render it', function() {
```
Comment (cschroed-usgs) [inline src/test/js/specs/AlertView_spec.js]: How does this test verify that the view hasn't rendered?

File: src/test/js/specs/AlertView_spec.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+		testView.show('alert-danger', 'Danger message');
+		expect(testView.template).toHaveBeenCalledWith({
+			message : 'Danger message',
+			alertClass : 'alert-danger'
+		});
+		expect($('#test-div .alert').length).toBe(1);
+	});
+
+	it('Expects that a second invocation of show removes the first alert and replaces with the seconde', function() {
```
Comment (cschroed-usgs) [inline src/test/js/specs/AlertView_spec.js]: seconde -> second

File: src/test/js/specs/AlertView_spec.js (hunk 1 above)
Comment (mbucknell) [inline src/test/js/specs/AlertView_spec.js]: It doesn't, but I can fix that! Thanks for the catch.

**Comment by cschroed-usgs** [comment]
Looks good! Nice tests. I'm glad we've got a tested alert solution. After those small fixes you are :+1: to merge.

**Pull request merged**